src/segy/file.py

```diff
@@ -49,6 +50,8 @@ class SegyFile:
             SEG-Y standard from the binary header.
```

Comment:
```suggestion
            SEG-Y standard from the binary header or header_overrides.
```

It looks like `header_overrides` will take precedence over `SegyFileSettings`